packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -123,6 +126,23 @@ describe("Test JsonRpcProvider Query Calls", () => {
       });
     });
 
+    describe("getInteractionByTesseract", () => {
+      it('should return the interaction by tesseract', async () => {
+        const interactions = await provider.getInteractionByTesseract(address, undefined, "0x0");
```

Comment:
height should be of type number.

Reply:
daf131566af57af4d2f8620381542beb36985b16